### ISSUE TITLE
Searches extension?

### ISSUE THREAD

**Issue by scotchale** [FreshRSS/Extensions#145]
Hi everybody, sorry my english: It would be good to have on the left sidebar the list of personal search keys and, after the feed update, Freshrss do a check of all the search keys. Every search key, listed in the sidebar and easly viewable, has the number of new feed that contains the relative key. Don't know if my propose is comprehensible, hope so!

**Comment by Alkarex** [comment]
I did not understand. Maybe in another language?
Vous pouvez essayer en français par exemple :-)

**Comment by math-GH** [comment]
search keys = user queries?

**Comment by scotchale** [comment]
an example in the sidebar (maybe better because always visible):
=  Main stream
    Favourites
>  User queries          (10)
    * query 1                (4)
    * query 2                (6)
v  My labels               
v  My category one    (50) 
v  My category two    (30)


Another point: is possible to have multiple words searched, in OR condition, in a single query?

**Comment by Alkarex** [comment]
> Another point: is possible to have multiple words searched, in OR condition, in a single query?

Yes and we also support parentheses and negation : `abc OR def`, `(ab cd) OR (ef gh)`

https://freshrss.github.io/FreshRSS/en/users/03_Main_view.html#with-the-search-field
